components/admin_console/file_upload_setting.jsx

```diff
@@ -32,10 +32,11 @@ export default class FileUploadSetting extends Setting {
             fileName: null,
             serverError: props.error,
         };
+        this.fileInputRef = React.createRef();
     }
 
     handleChange = () => {
-        const files = this.refs.fileInput.files;
+        const files = this.fileInput.current.files;
```

Comment:
shouldn't this be `this.fileInputRef`? I see it also in some other parts of the file

Reply:
Resolved it.